api/routes/index.js

```diff
@@ -11,6 +11,8 @@ const modify = require('../controllers/modify');
 const deleteUser = require('../controllers/delete');
 const insertUserData = require('../controllers/insertUserData');
 const getUserData = require('../controllers/getUserData');
+const addRoleToUser = require('../controllers/addRoleToUser');
```

Comment:
Perhaps being pedantic, but can we change the name to just "addRole"? The "ToUser" part is pretty self evident.